### PR TITLE
feat: add studio cli flow troubleshooting

### PR DESCRIPTION
**Pull request by ykethan**
#### Description of changes:
documents cli flow common issues and their remediation steps 
 
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [x] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/console/adminui/extend-cli.mdx [hunk 1]
```diff
@@ -1,50 +1,91 @@
...
+
+### Logging into Amplify Studio using Amplify CLI does not provide a Token or Prompt
+
+This behavior is observed when running a Amplify CLI headless pull command in the terminal, which opens a new tab in the browser with a Amplify Studio login screen. On logging into the Amplify Studio application we observe the terminal prompt is in a loading state.
+
+To resolve this issue,
+
+1. Exit from the Amplify CLI prompt
+1. If present, remove the empty folders created in your present directory (Ex: `amplify` or `amplify-backup`) from the previous pull
+1. Rerun the Amplify CLI headless pull command
```
Comment (josefaidt) [inline src/pages/console/adminui/extend-cli.mdx:67]:
```suggestion
2. If present, remove the leftover artifacts created in your current directory (Ex: `amplify` or `amplify-backup`) from the failed pull attempt
3. Rerun the Amplify CLI headless pull command
```

File: src/pages/console/adminui/extend-cli.mdx [hunk 2]
```diff
@@ -1,50 +1,91 @@
...
+2. Ability to easily replicate backend environments across AWS accounts and regions.
+3. Ability to easily integrate with CI/CD pipelines such as the pipeline offered in the Amplify console.
+
+All backend infrastructure-as-code definitions can be added to your project with the Amplify Command Line Interface (CLI).
+
+## Troubleshooting
+
+### Logging into Amplify Studio using Amplify CLI does not provide a Token or Prompt
+
+This behavior is observed when running a Amplify CLI headless pull command in the terminal, which opens a new tab in the browser with a Amplify Studio login screen. On logging into the Amplify Studio application we observe the terminal prompt is in a loading state.
```
Comment (josefaidt) [inline src/pages/console/adminui/extend-cli.mdx:61]:
```suggestion
This behavior is observed when running a Amplify CLI headless pull command (i.e. `amplify pull ... --yes`) in the terminal, which opens a new tab in the browser with a Amplify Studio login screen. On logging into the Amplify Studio application we observe the terminal prompt is in a loading state.
```

or without the ellipses (`...`)

File: src/pages/console/adminui/extend-cli.mdx [hunk 3]
```diff
@@ -1,50 +1,91 @@
...
+
+### Authentication tokens
+
+When using Amplify CLI, you may encounter the following error message:
+
+<Callout info>
+
+"Failed to receive expected authentication tokens. Error: [Error: 'jsonString' argument missing or empty]"
+
+</Callout>
```
Comment (josefaidt) [inline src/pages/console/adminui/extend-cli.mdx:77]: Can we move this to a fenced code block? 

```console
"Failed to receive expected authentication tokens. Error: [Error: 'jsonString' argument missing or empty]"
```

File: src/pages/console/adminui/extend-cli.mdx [hunk 4]
```diff
@@ -1,50 +1,91 @@
...
+
+To resolve this issue,
+
+1. Exit from the Amplify CLI prompt
+1. If present, remove the empty folders created in your present directory (Ex: `amplify` or `amplify-backup`) from the previous pull
+1. Rerun the Amplify CLI headless pull command
+
+### Authentication tokens
+
+When using Amplify CLI, you may encounter the following error message:
```
Comment (josefaidt) [inline src/pages/console/adminui/extend-cli.mdx:71]:
```suggestion
When using Amplify CLI you may encounter the following error message:
```

**Review by wrpeck — APPROVED** [review]
LGTM!

**Review by josefaidt — APPROVED** [review]

**Review by josefaidt — APPROVED** [review]

**Review by wrpeck — APPROVED** [review]
LGTM

**Review by petercwk — APPROVED** [review]
lgtm

**Review by katiegoines — APPROVED** [review]

File: src/pages/console/adminui/extend-cli.mdx [hunk 5]
```diff
@@ -1,50 +1,90 @@
 export const meta = {
   title: `Extend with the Amplify CLI`,
-  description: `Install the CLI without npm and use the CLI without an AWS account`,
+  description: `Install the CLI without npm and use the CLI without an AWS account`
```
Comment (cwoolum) [inline src/pages/console/adminui/extend-cli.mdx:3]: `without npm` - don't we recommend against not using NPM?

File: src/pages/console/adminui/extend-cli.mdx [hunk 6]
```diff
@@ -1,50 +1,90 @@
...
+2. Ability to easily replicate backend environments across AWS accounts and regions.
+3. Ability to easily integrate with CI/CD pipelines such as the pipeline offered in the Amplify console.
+
+All backend infrastructure-as-code definitions can be added to your project with the Amplify Command Line Interface (CLI).
+
+## Troubleshooting
+
+### Logging into Amplify Studio using Amplify CLI does not provide a Token or Prompt
+
+This behavior is observed when running a Amplify CLI `amplify pull --appId <appID> --envName <envName>` command in the terminal, which opens a new tab in the browser with a Amplify Studio login screen. On logging in, if you are not redirected back and observe the CLI being stuck in a loading state.
```
Comment (cwoolum) [inline src/pages/console/adminui/extend-cli.mdx:61]: This is a broken sentence

`On logging in, if you are not redirected back and observe the CLI being stuck in a loading state.`

File: src/pages/console/adminui/extend-cli.mdx [hunk 7]
```diff
@@ -1,50 +1,90 @@
...
+
+To resolve this issue,
+
+1. Delete the `config.json` file present at the location `~/.amplify/admin/`
+1. If present, remove the leftover artifacts created in your current directory (e.g. `amplify` or `amplify-backup`) from the failed pull attempt
+1. Rerun the Amplify CLI `amplify pull --appId <appID> --envName <envName>` command
+
+<Callout>
+
+Note, the content of `~/.amplify/admin/config.json` is utilized to store JWTs to vend temporary AWS credentials and should __NOT__ be shared.
```
Comment (cwoolum) [inline src/pages/console/adminui/extend-cli.mdx:88]: Can we be consistent on how we define bold text? We use `**` elsewhere but use `__` here

**Pull request merged**